src/services/twilio-help/twilio-command-help.js

```diff
@@ -31,6 +33,30 @@ class TwilioCommandHelp extends CommandHelp.default {
     return returnList.join('\n');
   }
 
+  // overriding to include help document link
+  examples(examples) {
+    if ((examples === undefined || !examples || examples.length === 0) && this.command.docLink === '') return '';
+    const listOfDetails = [];
+    if (examples !== undefined && examples.length !== 0) {
+      const body = util
+        .castArray(examples)
+        .map((a) => this.render(a))
+        .join('\n');
+      listOfDetails.push(chalk.bold(`EXAMPLE${examples.length > 1 ? 'S' : ''}`), indent(body, 2));
+    }
+    listOfDetails.join('\n\n');
+    const helpDoc = this.command.docLink || this.getHelpDocForTopic(this.command.id);
+    if (helpDoc !== undefined) {
+      listOfDetails.push(chalk.bold('MORE INFO'));
+      listOfDetails.push(indent(helpDoc, 2));
```

Comment:
`[Suggestion` -> Should a hyperlink here so the URL is clickable (Easier to navigate for any user! :) )

```
await cli.url(indent(helpDoc, 2), helpDoc);
```

Ref: Cli-ux > https://www.npmjs.com/package/cli-ux